gif.h

```diff
@@ -608,7 +620,11 @@ void GifWriteLzwImage(FILE* f, uint8_t* image, uint32_t left, uint32_t top,  uin
     fputc(0x21, f);
     fputc(0xf9, f);
     fputc(0x04, f);
-    fputc(0x05, f); // leave prev frame in place, this frame has transparency
+    if(transparent) {
```

Comment:
Ha-ha! :rofl: 
You just now forgot to add the argument to this function! :fox_face: 
![default](https://user-images.githubusercontent.com/6751442/40258336-e5108d66-5af9-11e8-972f-5a342705e50a.png)
Look carefully, dude! :wink: